thunder/benchmarks/targets.py

```diff
@@ -109,7 +111,7 @@ def deco(old_timer):
         @functools.wraps(old_timer)
         def timer():
             ret = old_timer()
-            benchmark.extra_info["max_allocated_memory_MB"] = torch.cuda.max_memory_allocated() / (1024 * 1024.0)
+            benchmark.extra_info[MAX_ALLOCATED_MEMORY_KEYWORD] = torch.cuda.max_memory_allocated() / (1024 * 1024.0)
```

Comment:
Why is this change needed?

Reply:
because in the `ThunderCompilerGraphBenchmarking` I query the key "max_allocated_memory_MB" in extra_info, so instead of using a hardcoded string, I change it to a constant str `MAX_ALLOCATED_MEMORY_KEYWORD`